content/posts/introducing-hippo.md

```diff
@@ -9,65 +9,122 @@ image: "images/twitter-card.png"
+The Deislabs team is really excited about WebAssembly. In fact, we've been
+experimenting and sharing our knowledge by developing new technologies in the
+WebAssembly space, such as [Krustlet](https://github.com/krustlet/krustlet),
+[WAGI](https://github.com/deislabs/wagi),
+[bindle](https://github.com/deislabs/bindle),
+[yo-wasm](https://github.com/deislabs/yo-wasm), and
+[wasi-experimental-http](https://github.com/deislabs/wasi-experimental-http).
 
-While experimenting with these new technologies, it became clear that we need to make it easy for developers to deploy and test out their new ideas.
+While experimenting with these new technologies, it became clear that we need
+to make it easy for developers to deploy and test out their new ideas.
 
 Today, we're thrilled to announce a new project.
 
 We call it Hippo, an Open Source self-hosted Platform as a Service (PaaS).
 
 ![hippo demo video](videos/hippo-intro.mp4)
 
-Hippo takes a fresh spin on the PaaS ecosystem, taking advantage of the technology WebAssembly brings to the space.
+Hippo takes a fresh spin on the PaaS ecosystem, taking advantage of the
+technology WebAssembly brings to the space.
 
-Hippo works like this: An application is packaged as a _bindle_. Bindles are collected together in a [bindle server](https://github.com/deislabs/bindle) that you can search. Hippo uses `bindle` under the hood for storing and organizing applications.
+Hippo works like this: An application is packaged as a _bindle_. Bindles are
+collected together in a [bindle server](https://github.com/deislabs/bindle)
+that you can search. Hippo uses `bindle` under the hood for storing and
+organizing applications.
 
-Hippo uses a concept called "Channels" to perform automatic deployments of your applications. More on that later.
+Hippo uses a concept called "Channels" to perform automatic deployments of your
+applications. More on that later.
 
-Using the `hippo` command line interface, you can upload new releases or prepare a bindle for local development.
+Using the `hippo` command line interface, you can upload new releases or
+prepare a bindle for local development.
 
-Hippo provides a web interface for users to register new accounts, access their applications, and create new environments for testing.
+Hippo provides a web interface for users to register new accounts, access their
+applications, and create new environments for testing.
 
-In the future, you can use this CLI to create applications, configure channels, gather logs, attach TLS certificates, and other commands you'd expect to use with a PaaS.
+In the future, you can use this CLI to create applications, configure channels,
```

Comment:
A bit of feedback that you should feel more than happy to ignore -- it feels a bit like up until this point, it's less of a consistent set of paragraphs, and more like individual sentences.

Personally I would try to group some of them, but then again, I tend to form larger than needed sentences, so...

Reply:
good point... I felt it looked awkward as well. I'll try to merge a bunch of them together and see how it looks.